eng/pipelines/tagger.yml

```diff
@@ -43,46 +39,21 @@ pool:
-  # https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/26284/Enabling-SBOM-For-Your-Component-Insertion-into-VS?anchor=1.-add-the-%27manifest-generator-task%27-to-your-pipeline
-  Packaging.EnableSBOMSigning: true
-  # Opt out of automatically injecting Codesign Validation into the pipeline. We run Codesign Validation as part of the Compliance pipeline.
+  # Opt out of automatically injecting Codesign Validation into the pipeline since this pipeline doesn't produce build artifacts.
   # See: https://aka.ms/gdn-injection
   runCodesignValidationInjection: false
-  # Suspend enforcement of NuGet Single Feed Policy. See:
-  # - https://aka.ms/nugetmultifeed
-  # - https://docs.opensource.microsoft.com/tools/nuget_security_analysis/nuget_security_analysis/
-  # - https://docs.opensource.microsoft.com/tools/cg/how-to/nuget-multifeed-configuration/
-  # - https://onebranch.visualstudio.com/OneBranch/_wiki/wikis/OneBranch.wiki/5205/TSG-Build-Broken-Due-to-Using-Multiple-Feeds?anchor=setting-nugetsecurityanalysiswarninglevel-in-cdp
-  NugetSecurityAnalysisWarningLevel: none
-  # Conditionally set the VsCommitId variable based on the VsCommitId parameter.
-  # If the parameter provides a value other than 'Default', it will be used. Otherwise, default to the commit ID of the VS repo.
-  # For conditional variables, see: https://stackoverflow.com/a/70954396/294804
-  # For Build.SourceVersion being set to VS repo commit ID, see: https://learn.microsoft.com/azure/devops/pipelines/repos/multi-repo-checkout?view=azure-devops#triggers
-  ${{ if eq(parameters.VsCommitId, 'Default') }}:
-    VsCommitId: $(Build.SourceVersion)
-  ${{ else }}:
-    VsCommitId: ${{ parameters.VsCommitId }}
-
-parameters:
-- name: VsCommitId
-  displayName: VS Commit ID
-  type: string
-  default: Default
-
+  Codeql.Enabled: false
+  
 ###################################################################################################################################################################
 # STEPS
 ###################################################################################################################################################################
 
 steps:
-- checkout: self
-  # Required for using Git commands in the subsequent tasks below. See:
-  # - https://stackoverflow.com/questions/56733922/fatal-could-not-read-password-for-https-organizationnamedev-azure-com-ter#comment108309839_56734304
-  # - https://learn.microsoft.com/azure/devops/pipelines/scripts/git-commands?view=azure-devops&tabs=yaml#allow-scripts-to-access-the-system-token
-  persistCredentials: true
-  # Changes Build.SourcesDirectory from $(Pipeline.Workspace)/s to $(Pipeline.Workspace)/project-system
-  path: project-system
-  # Hard-coded assumption that the commit referenced by the VS insertion is within the last 100 commits to main in the repo.
-  fetchDepth: 100
+- checkout: DotnetProjectSystem
+  persistCredentials: 'true'
+  path: 'project-system'
+  fetchDepth: 1000
```

Comment:
What does this do? The fetch below passes a depth of 100.

Reply:
This is the fetch in our github repo. Assuming that the last inserted commit from our branch would be in last 1000 commits. While we insert pretty regularly, I am not entirely sure about the delay in creation of tag in VS repo. Further, not entirely sure how far we would move when they release a patch. But assuming every even version being LTS and serviced, 1000 should also give us good coverage for servicing tags.

The 100 below is depth in VS repo tag we checkout. Assuming in last 100 commits we inserted at least once.